lib/assets/news-stylesheet.scss

```diff
@@ -115,86 +86,91 @@ footer {
+    line-height: 0;
+
+    a {
+        display: block;
+        padding: 0.6em 0.9em;
+        svg {
+            fill: rgba(0, 0, 0, 0.6);
+            width: 1rem;
+            height: 1rem;
+            cursor: pointer;
+        }
+    }
+}
+
+
+// Responsive Elements
 
 // Define all our responsive properties in terms of three parameters, then
 // include this mixin in the various media queries
-@mixin responsive-elements($body-size, $indent: 20.4, $dedent: -10.2) {
-    $real-indent: $paper-width * ($indent / 100);
-    $real-dedent: $paper-width * ($dedent / 100);
-
+@mixin responsive-elements($body-size, $indent, $dedent) {
     html {
         font-size: $body-size;
     }
 
     body {
-        padding-left: $real-indent;
-        padding-right: $real-indent;
+        margin-left: $indent;
+        margin-right: $indent;
     }
 
-    .article-information, .article-title, blockquote {
-        margin-left: $real-dedent;
-        margin-right: $real-dedent;
+    footer {
+        margin-left: -$indent;
+        margin-right: -$indent;
+    }
+
+    figure,
+    .article-information,
+    .article-title h1 {
+        margin-left: $dedent;
+        margin-right: $dedent;
     }
 
     .lede {
-        margin-left: $real-dedent;
+        margin-left: $dedent;
     }
+}
 
-    blockquote {
-        p, cite {
-            flex: calc(100% + #{$real-dedent});
-        }
+@include responsive-elements($body-size: 20px, $indent: 20vw, $dedent: -4.2vw);
 
-        .quote-decoration {
-            $size: calc(#{-$real-dedent} - 1rem);
-            flex: $size 0 0;
-            height: $size;
-            width: $size;
-        }
-    }
+@media (min-width: $spacious-min-width) {
+    @include responsive-elements($body-size: 24px, $indent: 24.2vw, $dedent: -4.2vw);
 
-    footer hr {
-        margin-left: -$real-indent;
-        margin-right: -$real-indent;
+    #default-share-actions {
+        right: 9vw;
     }
 }
 
-@include responsive-elements($body-size: 20px);
-
 @media (max-width: $compact-max-width) {
-    $blockquote-extra-indent: 10;
-    @include responsive-elements($body-size: 20px, $indent: 10, $dedent: 0);
-
-    blockquote {
-        $real-extra-indent: $paper-width * $blockquote-extra-indent / 100;
-        p, cite {
-            flex: calc(100% - #{$real-extra-indent});
-        }
-
-        .quote-decoration {
-            $size: calc(#{$real-extra-indent} - 1rem);
-            flex: $size 0 0;
-            height: $size;
-            width: $size;
+    @include responsive-elements($body-size: 20px, $indent: 15vw, $dedent: 0);
+
+    figure {
+        margin-left: -15vw;
+        margin-right: -15vw;
+        figcaption {
+            margin-left: 15vw;
+            margin-right: 15vw;
         }
     }
-}
 
-@media (min-width: $spacious-min-width) {
-    @include responsive-elements($body-size: 24px);
-}
-
-// Composite resolutions
-@media (device-width: 720px) and (device-height: 480px),
```

Comment:
Removing these will default composite resolutions to the `$compact-max-width` rules, which will give the body left and right margins of 15vw. Is that intentional?

Reply:
@ptomato Yes, that's intentional. 15vw at that screen size is small enough to not take up too much space, but large enough to create good visual margins + leave room for the sharing feature on the right side.

Reply:
@ptomato good catch -- I thought I had fixed that but didn't thoroughly test composite. Just added new styling for composite mode.